### PR TITLE
Change

### PR DESCRIPTION
**Pull request by hector0513**
Clipboard - 7 de noviembre de 2019 17_31

### CODE REVIEW

**Comment by hector0513** [comment]
Image Pasted at 2019-11-7 17-27
Image Pasted at 2019-11-7 17-07

**Review by Connoropolous — CHANGES_REQUESTED** [review]
Nice. If you don't use UpperRightMenu, then just delete that component as well. 

Can you hide the Notification and Search icons for now, until we implement those features? Leave them in the code just comment them out?

**Comment by hector0513** [comment]
ready
i deleted  UpperRightMenu

File: src/components/Empty/Header.js [hunk 1]
```diff
@@ -72,16 +79,14 @@ export default class Header extends React.Component{
                     </div>
                 </div>
                 <div className="status2">
-                    <img src="img/search-line.svg"/>
                     <img src="img/notebook-line.svg" onClick={this.clickBook}/>
```
Comment (Connoropolous) [inline src/components/Empty/Header.js:86]: Can you convert this to use the Icon component?

Comment (Connoropolous) [reply src/components/Empty/Header.js:86]: Should be very simple. Very small change

**Review by Connoropolous — CHANGES_REQUESTED** [review]
One more follow-up I noticed. Thanks for your patience.

**Comment by hector0513** [comment]
this ?

File: src/components/Empty/Header.js (hunk 1 above)
Comment (hector0513) [reply src/components/Empty/Header.js:86]: ok

Comment (hector0513) [reply src/components/Empty/Header.js:86]: ready

**Review by Connoropolous — APPROVED** [review]
Awesome!

**Pull request merged**